pkg/logger/logger.go

```diff
@@ -0,0 +1,59 @@
+package logger
+
+import (
+	"time"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	logger *zap.Logger
+	level  zap.AtomicLevel
+)
+
+func init() {
```

Comment:
I'd not use `init()`, instead `new()` to match conventions
```suggestion
func new() {
```

Reply:
`init()` method is a special method which is being called automatically during an app initialisation. The idea is to setup global logger that can be used in other part of the app and have it configured before it is being used.